content/news/2022.pml

```diff
@@ -54,6 +54,30 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item" id="summit-horizon">
+    7 Spetember 2022: Thanks Horizon Robotics for sponsoring the seL4 Summit 2022
+</div>
+<div>
+  <p>
+    <a href="https://horizon.ai/">
+    <img src="/Foundation/Membership/LOGOS/Horizon_Robotics.svg" style="width: 20%;  padding-left:10px;  float:right"
+              alt="Horizon Robotics logo">
+        </a>
+  </p>
+  <p>
+    The seL4 Foundation thanks <a href="https://horizon.ai/">Horizon Robotics</a> for becoming a seL4 Summit 2022 sponsor.
```

Comment:
add link to summit page + add "Bronze" ("for becoming a Bronze sponsor for the seL4 summit 2022")